packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1017,7 +1017,15 @@ abstract class Element<T extends Widget> implements BuildContext {
 }
 
 class ErrorWidget extends LeafRenderObjectWidget {
-  RenderBox createRenderObject() => new RenderErrorBox();
+  ErrorWidget(Object exception) : message = _stringify(exception), super(key: new GlobalKey());
```

Comment:
If you're just trying to force this to not sync with the previous widget, you can create a new ObjectKey(new Object()).  That's less likely to cause problems if, for example, the global key table is corrupt.